src/main/scripts/kafka-topology-builder.sh

```diff
@@ -4,4 +4,4 @@ KAFKA_TOPOLOGY_BUILDER_HOME=/usr/local/kafka-topology-builder
 KTB_JAR=$KAFKA_TOPOLOGY_BUILDER_HOME/bin/kafka-topology-builder.jar
 JAVA_PATH=java
 
-$JAVA_PATH -jar $KTB_JAR "$@"
+$JAVA_PATH $JVM_OPTIONS -jar $KTB_JAR "$@"
```

Comment:
I wonder, should we have a default value for this variable?

Reply:
As the tool is used by some people as well outside the scope of docker I think it would be better to namespace the variable.

What about naming it _$KTB_JVM_OPTIONS_ ?

Reply:
I took the liberty of adding the commits myself, hopefully, this is ok for you. I will move forward and merge it soon as I find this very valuable.